README.md

```diff
@@ -103,26 +103,40 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Building
 -
-To build the manual you will need go-md2man.
+To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag.
+
+### Building in a container
+If your system has a `docker` command and can run Linux containers,
 ```sh
-$ sudo apt-get install go-md2man
+$ make binary # Or (make all) to also build documentation, see below.
 ```
-To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag. Also, make sure to clone the repository in your `GOPATH` - otherwise compilation fails.  
+(Note that the created executable is a Linux executable, and depends on dynamic libraries which may only be available only in a container of a similar Linux distribution.)
```

Comment:
So what is the long term vision for skopeo? Will it always be a Linux CLI tool? There is not intention to make it work on other OS types?